modules/cij/runner.py

```diff
@@ -416,7 +430,8 @@ def trun_to_junitfile(trun, fpath=None):
                     doc_failure = doc.createElement("failure")
                     doc_failure.setAttribute(
                         "message",
-                        "not executed" if tcase.rcode is None else "test failed"
+                        "not executed" if tcase.rcode is None
+                        else "test failed"
```

Comment:
I would personally indent the `else` to align with the `if`

Reply:
I am going with:
```
 "not executed" if tcase.rcode is None else
 "test failed"
```
Linters to not appreciate the long indent to match up to ``if``